java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -642,6 +643,67 @@ void testReadParquetContainsDecimalData() {
     }
   }
 
+  @Test
+  void testReadAvro() {
+    AvroOptions opts = AvroOptions.builder()
+            .includeColumn("bool_col")
```

Comment:
Nit: Continuation indent is 4 spaces, applies to the rest of the changes in this file.

Reply:
Done